src/connectors/infinite-hits/connectInfiniteHits.js

```diff
@@ -115,7 +118,10 @@ export default function connectInfiniteHits(renderFn, unmountFn) {
           results.hits = escapeHits(results.hits);
         }
 
-        hitsCache = [...hitsCache, ...results.hits];
+        if (lastReceivedPage < state.page) {
```

Comment:
we'll need to be careful with this line if we ever allow "scrolling up" as in #1972

Reply:
Do you think it's blocking for now?

Reply:
I think for now it's fine, we just need to keep it in mind if we ever allow scrolling to go the other direction

Reply:
Ok thanks :)